samples/OpenIdConnect.AzureAdSample/project.json

```diff
@@ -8,7 +8,7 @@
     "Microsoft.Extensions.Configuration.EnvironmentVariables": "1.0.0-*",
     "Microsoft.Extensions.Configuration.UserSecrets": "1.0.0-*",
     "Microsoft.Extensions.Logging.Console": "1.0.0-*",
-    "Microsoft.IdentityModel.Clients.ActiveDirectory": "3.9.302261508-alpha"
+    "Microsoft.IdentityModel.Clients.ActiveDirectory": "3.11.305310302-alpha"
   },
   "frameworks": {
     "net451": { },
```

Comment:
no, the major, minor, and patch are all ints, it only gets into strict alphabetical in the prerelease tag.